uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Page/UserMenu.php

```diff
@@ -1,102 +1,13 @@
 <?php
 
-use CRM_HRCore_CMSData_PathsFactory as CMSPathsFactory;
-
 class CRM_HRCore_Page_UserMenu extends CRM_Core_Page {
```

Comment:
Just to confirm, we still need this page because on the admin you'll load the markup using Ajax, right?

Reply:
Yep